tests/conftest.py

```diff
@@ -105,3 +104,24 @@ def dummy_agent_tim(test_packages_filesystem) -> Path:
         raise ValueError(msg)
     os.chdir(agent.name)
     return True
+
+
+@pytest.fixture
+def dummy_agent_default(test_packages_filesystem) -> Path:
```

Comment:
see dummy agent tim which already exists

Reply:
i didn't want to break existing tests, and needed a agent that had --no-publish

Reply:
ok then maybe make the testname specify that, as why default if that makes sense